src/stories/6_Addons.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Overview/Addons" />
+
+# Addons
+
+The RoleModel Design System provides a few addons for integrating Third-Party tools with your application.
+
+## Tom Select
+
+[Tom Select](https://tom-select.js.org/) is a dynamic, framework agnostic, and lightweight (~16kb gzipped) `<select>` UI control. With autocomplete and native-feeling keyboard navigation, it's useful for tagging, contact lists, country selectors, and so on.
+
+If you application uses Tom Select, you can import the styles from Tom Select along with the Design Systems tokens applied to it.
+```scss
+@import '@rolemodel/rolemodel-design-system/dist/scss/rolemodel-design-system'
+@import '@rolemodel/rolemodel-design-system/dist/scss/addons/tom-select';
+```
+
+## Modal and Panel
+
+Modal and Panel provide styles that are intended to accompany the rails configuration and javascript implemented by [RoleModel Rails Modal and Panel](https://github.com/RoleModel/rolemodel_rails/tree/master/lib/generators/rolemodel/modals)
+
+If you application uses this implementation, you can import the styles via:
+```scss
+@import '@rolemodel/rolemodel-design-system/dist/scss/rolemodel-design-system'
+@import '@rolemodel/rolemodel-design-system/dist/scss/addons/modal-and-panel';
```

Comment:
@Jeremy-Walton Suggest renaming this to `rails-modal-and-panel` if we want to keep it specific. For a more generic name that could eventually encompass additional components perhaps just `rails-components`...

Reply:
I'll add the rails prefix.

A generic `rails-components` might be useful in the future, but I don't anticipate a lot of "rails" specific components being added to the design system. Most of the time our components are very generic and not tied to rails. Modal and panel are because of the JS implementation behind them.

Eventually, I could see an addon for "Shoelace token mapping" and then maybe a `shoelace-components` section, but it really depends on the adoption of that and how generic we find they are.